assets/js/admin-scripts.js

```diff
@@ -1853,19 +1853,33 @@ jQuery(document).ready(function ($) {
 				var email       = wrapper.find('input[name="additional-email"]').val();
 				var primary     = wrapper.find('input[name="make-additional-primary"]').is(':checked');
 				var nonce       = wrapper.find('input[name="add_email_nonce"]').val();
+				var forceLink 	= false;
+
+				if ( e.target.id === 'add-customer-email-force' ){
+					forceLink = true;
+				}
 
 				var postData = {
 					edd_action:  'customer-add-email',
 					customer_id: customer_id,
 					email:       email,
 					primary:     primary,
+					forceLink:   forceLink,
```

Comment:
Our preferred variable format is snake case, not camel case. Could we change that to `force_link`, so that we don't have to guess what version we are using in cases like this where we now have 2 different cases of how `forceLink and force_link` mean the same thing.
`$force_link  = 'true' === $args['forceLink'] ? true : false;`